x-pack/plugins/apm/server/routes/alerts/rule_types/error_count/register_error_count_rule_type.ts

```diff
@@ -86,6 +87,12 @@ export function registerErrorCountRuleType({
       minimumLicenseRequired: 'basic',
       isExportable: true,
       executor: async ({ params: ruleParams, services, spaceId }) => {
+        const predefinedGroupby: string[] = [SERVICE_NAME, SERVICE_ENVIRONMENT];
+
+        const allGroupbyFields: string[] = [
+          ruleParams.groupBy ?? predefinedGroupby,
+        ].flat();
```

Comment:
We are adding new rule parameter `groupBy`. I added this check to support existing rules when upgrading (without modifying the rule in UI). The existing rules will have `groupBy` undefined so I am adding here predefined group-by fields in that case.
We could always add them, but then we need to use `Set` to filter out duplicates. Because we are always adding default group-by fields in the rule parameters when the rule is saved in UI.
```
new Set([...ruleParams.groupBy, ... predefinedGroupby])
```
Regarding the API, could you tell me how can I test with the API?